.vscode/settings.json

```diff
@@ -1,25 +1,27 @@
 {
-  "typescript.tsdk": "./node_modules/typescript/lib",
+  "editor.insertSpaces": true,
+  "editor.rulers": [80],
+  "editor.tabCompletion": true,
   "editor.tabSize": 2,
   "editor.trimAutoWhitespace": true,
-
   "files.exclude": {
+    "**/.DS_Store": true,
     "**/.git": true,
-    "**/.svn": true,
     "**/.hg": true,
+    "**/.svn": true,
     "**/CVS": true,
-    "**/.DS_Store": true,
-    "packages/*/lib": true,
-    "packages/*/lib6": true,
+    ".nyc_output": true,
     "coverage": true,
-    ".nyc_output": true
+    "packages/*/lib": true,
+    "packages/*/lib6": true
   },
-  "files.trimTrailingWhitespace": true,
   "files.insertFinalNewline": true,
-
-  "tslint.ignoreDefinitionFiles": true,
+  "files.trimTrailingWhitespace": true,
+  "tslint.autoFixOnSave": true,
```

Comment:
I won't enable this option by default as `tslint --fix` does not work reliably AFAIK.